album/KSLA-1001~50.toml

```diff
@@ -0,0 +1,2355 @@
+
+[[discs.tracks]]
+title = "時を刻む唄 (100-200+tanigon remix)"
+artist = "Lia"
+
+[[discs.tracks]]
+title = "星屑(Kouki Izumi Remix)"
+artist = "霜月はるか"
+
+[[discs.tracks]]
+title = "Little Busters! (Nhato Remix)"
+artist = "Rita"
+
+[[discs.tracks]]
+title = "Alicemagic(Tsukasa Remix)"
+artist = "Rita"
+
+[[discs.tracks]]
+title = "Philosophyz(MUZIK SERVANT Remix)"
+artist = "水谷瑠奈（NanosizeMir）"
+
+[[discs.tracks]]
+title = "Rewrite (DJ Shimamura's Drumscape Remix)"
+artist = "サイキックラバー"
+
+[[discs.tracks]]
+title = "鳥の詩（PandaBoY Remix）"
+artist = "Lia"
+
+[[discs]]
+title = "Key＋VOCALOID Best selection vol.1 より"
+artist = "初音ミク"
+catalog = "KSLA-1030"
+
+[[discs.tracks]]
+title = "恋文"
+
+[[discs.tracks]]
+title = "鶏肉の唄"
+
+[[discs.tracks]]
+title = "オーバー"
+
+[[discs.tracks]]
+title = "鈴の密かな恋の唄"
+
+[[discs.tracks]]
+title = "雨のち晴れ"
+
+[[discs.tracks]]
+title = "Farewell song"
+
+[[discs.tracks]]
+title = "raison"
+
+[[discs.tracks]]
+title = "CANOE"
+
+[[discs.tracks]]
+title = "灰色の羽根"
+artist = "鏡音リン、初音ミク"
+
+[[discs.tracks]]
+title = "Brave Song"
+
+[[discs.tracks]]
+title = "一番の宝物"
+
+[[discs]]
+title = "KeyCLASSIC・Key Acoustic Arrange Album 『Humans pleasure』 他より"
+catalog = "KSLA-1031"
+
+[[discs.tracks]]
+title = "Last regrets～Classic～"
+artist = "彩菜"
+
+[[discs.tracks]]
+title = "青空～Classic～"
+artist = "Lia"
+
+[[discs.tracks]]
+title = "メグメル～Classic～"
+artist = "riya"
+
+[[discs.tracks]]
+title = "時を刻む唄～Classic～"
+artist = "Lia"
+
+[[discs.tracks]]
+title = "Little Busters!～Classic～"
+artist = "Rita"
+
+[[discs.tracks]]
+title = "一番の宝物～yui.ver Classic～"
+artist = "LiSA"
+
+[[discs.tracks]]
+title = "Philosophyz～Classic～"
+artist = "水谷瑠奈（NanosizeMir）"
+
+[[discs.tracks]]
+title = "Harvest～Classic～"
+artist = "多田葵"
+
+[[discs.tracks]]
+title = "Little Busters!"
+artist = "Rita"
+
+[[discs.tracks]]
+title = "届けたいメロディ"
+artist = "北沢綾香"
+
+[[discs.tracks]]
+title = "星の舟"
+artist = "Lia"
+
+[[discs.tracks]]
+title = "Farewell Song"
+artist = "Lia"
+
+[[discs.tracks]]
+title = "涙色の翼"
+artist = "北沢綾香"
+
+[[discs]]
+title = "I've × Key Collaboration Album・ビジュアルアーツ・I've驚愕の丸秘楽曲集・Rewrite Arrange Album 'Selene' 他より"
+catalog = "KSLA-1032"
+
+[[discs.tracks]]
+title = "Boys be Smile ～Remix～"
+artist = "鈴湯"
+
+[[discs.tracks]]
+title = "Farewell song"
+artist = "IKU"
+
+[[discs.tracks]]
+title = "Last regrets"
+artist = "RINA"
+
+[[discs.tracks]]
+title = "Saya's song"
+artist = "Larval Stage Planning"
+
+[[discs.tracks]]
+title = "Little Busters!"
+artist = "柚子乃"
+
+[[discs.tracks]]
+title = "青空"
+artist = "Lily on the FIeLD"
+
+[[discs.tracks]]
+title = "隠しトラック（Last regrets 仮歌ver）"
+artist = "高瀬一矢"
+
+[[discs.tracks]]
+title = "録りの詩"
+artist = "高瀬一矢"
+
+[[discs.tracks]]
+title = "風邪の辿り着く場所"
+artist = "高瀬一矢"
+
+[[discs.tracks]]
+title = "Philosophyz (溝口ゆうま remix)"
+artist = "水谷瑠奈（NanosizeMir）"
+
+[[discs.tracks]]
+title = "サンブライト (ALR REMIX)"
+artist = "北沢綾香"
+
+[[discs.tracks]]
+title = "Twinkle starlight HardCore Remix"
+artist = "佐咲紗花"
+
+[[discs]]
```

Comment:
All titles of tracks in this disc miss a space before left bracket. (recommended (? (since next disc all have the space